ddtrace/internal/remoteconfig/worker.py

```diff
@@ -31,19 +33,53 @@ def get_poll_interval_seconds():
 
 
 class RemoteConfigWorker(periodic.PeriodicService):
+    """Remote configuration worker.
+
+    This implements a finite-state machine that allows checking the agent for
+    the expected endpoint, which could be enabled after the client is started.
+    """
+
     def __init__(self):
         super(RemoteConfigWorker, self).__init__(interval=get_poll_interval_seconds())
         self._client = RemoteConfigClient()
         log.debug("RemoteConfigWorker created with polling interval %d", get_poll_interval_seconds())
+        self._state = self._agent_check
 
-    def periodic(self):
+    def _agent_check(self):
+        # type: () -> None
+        data = agent._healthcheck()
+        if data:
+            endpoints = data.get("endpoints", [])
+            if endpoints and (
+                REMOTE_CONFIG_AGENT_ENDPOINT in endpoints or ("/" + REMOTE_CONFIG_AGENT_ENDPOINT) in endpoints
+            ):
+                self._state = self._online
+                return
+
+        log.warning(
+            "Agent is down or Remote Config is not enabled in the Agent\n"
+            "Check your Agent version, you need an Agent running on 7.39.1 version or above.\n"
```

Comment:
I would suggest to reduce this message and just redirect them to the setting page or the RC documentation (should be released soon) to avoid outdated comments. 
For example: the remote config key would be removed soon when normal keys can be marked as "rc enabled".